script/DeployMocks.s.sol

```diff
@@ -0,0 +1,41 @@
+// SPDX-License-Identifier: BUSL 1.1
+// Valorem Labs Inc. (c) 2023.
+pragma solidity 0.8.16;
+
+import "forge-std/Script.sol";
+
+// import "../src/TokenURIGenerator.sol";
+import "test/mocks/MockUSDC.sol";
+import "test/mocks/MockWETH.sol";
+import "test/mocks/MockGMX.sol";
+
+
+// Salt for mocks
+
+contract DeployMocksScript is Script {
+        uint256 constant SALT = 0x2023091301;
+//  Mock USDC Deployed to address: 0x37FeA693DC8C1CeA5244Ca2494762328e56dd959
+//  Mock WETH Deployed to address: 0x1D621b431bF56a3fd49339FF0f9ea9F5B8933C1d
```

Comment:
Can we add MockWBTC here? This will give us a mock with 8 decimals, which will be helpful.

```
// SPDX-License-Identifier: BUSL 1.1
// Valorem Labs Inc. (c) 2023.
pragma solidity 0.8.16;

import "./MockERC20.sol";

/// @notice Mock for Wrapped Bitcoin
contract MockWBTC is MockERC20 {
    constructor() MockERC20("Wrapped BTC", "WBTC", 8) {}
}
```